src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -76,33 +101,23 @@
     <Compile Include="System\ComponentModel\ReflectTypeDescriptionProvider.ReflectedTypeData.cs" />
     <Compile Include="System\ComponentModel\RefreshEventArgs.cs" />
     <Compile Include="System\ComponentModel\RefreshEventHandler.cs" />
-    <Compile Include="System\ComponentModel\SByteConverter.cs" />
-    <Compile Include="System\ComponentModel\SingleConverter.cs" />
-    <Compile Include="System\ComponentModel\StringConverter.cs" />
-    <Compile Include="System\ComponentModel\TimeSpanConverter.cs" />
-    <Compile Include="System\ComponentModel\TypeConverter.cs" />
-    <Compile Include="System\ComponentModel\TypeConverterAttribute.cs" />
     <Compile Include="System\ComponentModel\TypeDescriptor.cs" />
     <Compile Include="System\ComponentModel\TypeDescriptionProvider.cs" />
     <Compile Include="System\ComponentModel\TypeDescriptionProviderAttribute.cs" />
-    <Compile Include="System\ComponentModel\TypeListConverter.cs" />
-    <Compile Include="System\ComponentModel\UInt16Converter.cs" />
-    <Compile Include="System\ComponentModel\UInt32Converter.cs" />
-    <Compile Include="System\ComponentModel\UInt64Converter.cs" />
     <Compile Include="System\ComponentModel\WeakHashtable.cs" />
     <Compile Include="System\ComponentModel\Design\IDictionaryService.cs" />
     <Compile Include="System\ComponentModel\Design\IExtenderListService.cs" />
     <Compile Include="System\ComponentModel\Design\ITypeDescriptorFilterService.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'==''">
     <ProjectReference Include="..\..\System.ComponentModel.Primitives\src\System.ComponentModel.Primitives.csproj">
```

Comment:
For now - the reference assembly has not been updated yet in master, so 4.1.0 that's available via nuget doesn't have the surface area we need